.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+Thank you for opening a Pull Request!
+
+Please provide answers to the following questions, delete examples.
```

Comment:
Is this two separate things we want users to do?  The phrasing is a bit confusing.  I suggest making it a bulleted list of tasks.

Reply:
How about:

Please provide answers to the following questions. Use the provided examples as a guide, delete them before submitting.

Reply:
Maybe also say "Keep the relevant headers marked with ##".

Reply:
Done